src/bepasty/static/app/js/utils.js

```diff
@@ -8,6 +8,35 @@ $(function () {
         });
     });
 
+    // Show a edit dialog box when trying to edit metadata
```

Comment:
Show an edit dialog box when trying to edit metadata.

Reply:
fixed. with "an edit" => "a modify"